src/js/auth.js

```diff
@@ -0,0 +1,65 @@
+const { nanoid } = 'nanoid';
+
+const auth = () => {
+  const getSession = () => {
+    const sessionData = sessionStorage.getItem('moshoodabdullahi-todo-list-app-session');
+    return sessionData ? JSON.parse(sessionData) : null;
+  };
+
+  const getLocalStorageUser = (userEmail) => {
+    const userArray = JSON.parse(localStorage.getItem('userArray')) || [];
```

Comment:
This key should be defined as a constant so that you can reuse it. Also, let's use a more complicated/specific key name.

Reply:
Done